Source/SwiftLintCore/Reporters/SummaryReporter.swift

```diff
@@ -44,6 +44,7 @@ private extension TextTable {
                 return true
             }
             if count1 == count2 {
+                // swiftlint:disable:next shorthand_argument
                 return $0 < $1
```

Comment:
We could use named arguments here to really fix the violation. Also the lines 41 and 42 can be improved by omitting the `keys`.